src/main/resources/l10n/JabRef_de.properties

```diff
@@ -2047,14 +2047,14 @@ Fetching_info_based_on_DOI=Rufe_Informationen_basierend_auf_DOI_ab
 Follow_DOI_or_URL_link_and_try_to_locate_PDF_full_text_document=Folge_DOI/URL-Link_um_ein_PDF-Volltextdokument_zu_finden
 Get_BibTeX_data_from_DOI=Hole_BibTeX-Daten_durch_DOI
 Left_entry=Linker_Eintrag
-Merge_entry_with_DOI_information=Eintrag_mit_DOI-Informationen_zusammenführen
+Merge_entry_with_DOI/ISBN_information=Eintrag_mit_DOI-Informationen_zusammenführen
 No_information_added=Keine_Informationen_hinzugefügt
 Opening_dialog=Öffne_Dialog
 Original_entry=Originaleintrag
 Replace_original_entry=Ersetze_Originaleintrag
 Right_entry=Rechter_Eintrag
 Select_at_least_one_entry_to_manage_keywords.=Es_muss_mindestens_ein_Eintrag_ausgewählt_sein_um_Stichworte_verwalten_zu_können.
-Updated_entry_with_info_from_DOI=Eintrag_wurde_mit_DOI-Information_aktualisiert.
+Updated_entry_with_info_from_DOI/ISBN=Eintrag_wurde_mit_DOI-Information_aktualisiert.
```

Comment:
Yes, that was my main reason for not putting "Ready for review". I'll sort it out eventually.